### DISCUSSION TITLE
SSO logon on MSSQL connection

### DISCUSSION THREAD

**Discussion by Taenzer70** [sqlpage/SQLPage#74] [Q&A]
When connecting to an MSSQL database, is it possible to log in with the Windows user instead of the SQL login (user/pass)?

**Comment by lovasoa** [comment]
Hello and thanks for opening this thread! 

I am not a windows or microsoft expert, so excuse me if my response is completely off, but doesn't `Windows Login` just mean that you provide your windows username and password to log in to the database ? Or is the authentication completely different at the protocol level ?

**Reply by lovasoa** [reply]
Ok, I answered too fast, a quick search informs me that I am wrong, and the authentication is indeed different at the protocol level. Unfortunately, it is not yet supported in sqlpage.

**Reply by Taenzer70** [reply]
What a pity. But that might explain why I couldn't get it to work.
